lib/commands/new.js

```diff
@@ -85,8 +85,32 @@ module.exports = async (cwd, dir, ctx) => {
         'directories'
     ];
 
+    const isDep = (name) => finalPkg.dependencies.hasOwnProperty(name);
+    const isDevDep = (name) => finalPkg.devDependencies.hasOwnProperty(name);
     const cmp = (x, y) => {
 
+        // Full-coverage here depends on input package file having keys to trigger
+        // all possible paths. The boilerplate's doesn't
+        // Only cases tested by package.json's entire set of keys is all false
+        // and half-true both ways (both deps or both dev deps).
+        /* $lab:coverage:off$ */
+        if (
+            (isDep(x.key) && isDep(y.key)) ||
+            (isDevDep(x.key) && isDevDep(y.key))
+        ) {
+            // boilerplate package.json deps are already alphabetical,
+            // so, the if clause is always true, meaning its only half-covered b/c false branch not tripped
+            // (this registers as missing coverage in node 12, but not 10)
+            // and the else branch is uncovered (never reached)
```

Comment:
This is the kind of thing that could be further covered by a unit test, but I'm okay pushing that off.  As mentioned in the comment, we also may take a different approach here altogether in the near future.